fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java

```diff
@@ -504,6 +504,8 @@ public class SessionVariable implements Serializable, Writable {
 
     public static final String JDBC_ORACLE_NULL_PREDICATE_PUSHDOWN = "jdbc_oracle_null_predicate_pushdown";
 
+    public static final String JDBC_CAST_EXPR_PUSHDOWN = "jdbc_cast_expr_pushdown";
```

Comment:
```suggestion
    public static final String JDBC_CAST_EXPR_PUSHDOWN = "enable_jdbc_push_down_cast_expr";
```

So that we can unify the variable name like `enable_jdbc_push_down_xxx`